File: BACKEND/Service/Models/CompanyModel.cs
namespace Service.Models
{
    public class CompanyModel
    {
        public Guid CompanyId { get; set; }

        public string CompanyName { get; set; } = null!;

        public string? Address { get; set; }

        public string? Email { get; set; }

        public string? Logo { get; set; }

        public string? Phone { get; set; }

        public string? Website { get; set; }

        public bool IsDeleted { get; set; } = false;
    }
}

File: FrontEnd/src/app/data/company/company.model.ts
export class Company {
	companyId?: string;
	companyName?: string;
	address?: string;
	email?: string;
	phone?: string;
	website?: string;
	isDeleted?: boolean;
	isActived?: boolean;
	logo?: string;
}

export class CompanyAddModel {
	companyName?: string;
	address?: string;
	email?: string;
	phone?: string;
	website?: string;
	isDeleted?: boolean;
	isActived?: boolean;
	logo?: string;
}


export class CompanyUpdateModel {
	companyName?: string;
	address?: string;
	email?: string;
	phone?: string;
	website?: string;
	isDeleted?: boolean;
	isActived?: boolean;
	logo?: string;
}
